### DISCUSSION TITLE
Force Index

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#382] [Indicators]
Community discussion about [Force Index](https://dotnet.stockindicators.dev/indicators/ForceIndex/#content)

**Comment by paramjit2k** [comment]
Will it work for Nifty Bank Index in India? It is working in trading view.

**Reply by DaveSkender** [reply]
Sure.  Any OHLCV quote history can generate a Force Index.  Checkout our demo site: https://stock-charts.azurewebsites.net
It's showing NASDAQ 100 Index.  You can add Force index to see it there too.

**Reply by DaveSkender** [reply]
> You can add Force index to see it there too.

Oops.  I take that back.  I haven't added Force Index as an option on the demo chart site yet.  In any case, it should work fine.

**Reply by paramjit2k** [reply]
it is showing ZERO in the ForceIndex value for all the candles. I have checked it with like 500 candles.

**Reply by paramjit2k** [reply]
> > You can add Force index to see it there too.
> 
> Oops. I take that back. I haven't added Force Index as an option on the demo chart site yet. In any case, it should work fine.

[Book1.xlsx](https://github.com/DaveSkender/Stock.Indicators/files/9912092/Book1.xlsx)

**Reply by DaveSkender** [reply]
Check your input `quotes` to make sure that both `Close` and `Volume` values are present.  If there's no `Volume` data, you'd get zeros.

**Reply by paramjit2k** [reply]
it worked like a miracle. Thanks, Dave.
